### PR TITLE
Final Review for chapter 4

### PR DESCRIPTION
**Pull request by anajsana**
Adding the changes that came from the final review during Chapter 4 meeting. During the call and the next two weeks, the changes were included using [this doc](https://github.com/todogroup/ospology/edit/main/ospo-book/content/en/04-chapter.md)for chapter intro and this doc for [activity table](https://drive.google.com/open?id=12wqqdCLgkTIor_onHvhCtLPE-Tsn44k2oWVxnisb6TM&authuser=0)

Contributors in this PR:

- Alice Sowerby
- Cornelius Schumacher
- Gergely Csatari
- Ana Jiménez
- Ulises Gascon
- Jan van den Berg

These people are already acknowledged in the OSPO Book contributors section.

Thank you!

### CODE REVIEW

File: ospo-book/content/en/04-chapter.md [hunk 1]
```diff
@@ -53,38 +50,88 @@ across various organizational divisions to align both technical and non-technica
...
+| Qualification of contributors          |                | Making sure that contributors have the knowledge and skills they need to act on behalf of the company in the public space of an open source project.               |
+
+| Leadership Stage |
+| --- |
+
+| Activities                                                                  | Value for OSPO | Value for Org                                                                                                                                                               |
+| --------------------------------------------------------------------------- | -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Open sourcing previously proprietary projects                               |                | Get the benefits of strategic involvement in and with open source                                                                                                           |
```
Comment (CsatariGergely) [inline ospo-book/content/en/04-chapter.md:98]:
```suggestion
| Open sourcing previously proprietary projects                               | Build open source community leadership capabilities in the organization | Get the benefits of strategic involvement in and with open source                                                                                                           |
```

File: ospo-book/content/en/04-chapter.md [hunk 2]
```diff
@@ -53,38 +50,88 @@ across various organizational divisions to align both technical and non-technica
...
+| Establish an “upstream first” policy                                        |                | Lead open source projects and make them part of the primary value creation of the organization                                                                              |
```
Comment (CsatariGergely) [inline ospo-book/content/en/04-chapter.md:99]:
```suggestion
| Establish an “upstream first” policy                                        | The OSPO can lean on the established leaders of the company open source projects               | Lead open source projects and make them part of the primary value creation of the organization                                                                              |
```

File: ospo-book/content/en/04-chapter.md [hunk 3]
```diff
@@ -53,38 +50,88 @@ across various organizational divisions to align both technical and non-technica
...
+| Supporting autonomy of contributors and maintainers of open source projects |                | Having people being dedicated to only open source work, the organization can strategically strengthen important open source projects in the most organic and effective way. |
```
Comment (CsatariGergely) [inline ospo-book/content/en/04-chapter.md:100]:
```suggestion
| Supporting autonomy of contributors and maintainers of open source projects | Allows the scalability of open source leadership and creates "open source" champions in the company               | Having people being dedicated to only open source work, the organization can strategically strengthen important open source projects in the most organic and effective way. |
```

File: ospo-book/content/en/04-chapter.md [hunk 4]
```diff
@@ -53,38 +50,88 @@ across various organizational divisions to align both technical and non-technica
...
+
+> Recommendation
+>
+> 1. SBOMs Compliance Ready: Ensure that all software components are documented through Software Bill of Materials (SBOMs). This documentation helps in quickly identifying potentially compromised components once a vulnerability is disclosed.
+> 
+> 2. Automation Security Checks: Implement automated security checks, such as the OpenSSF Security Scorecard, to continuously evaluate the security posture of projects. This proactive measure can highlight vulnerabilities or anomalies that merit further investigation.
```
Comment (CsatariGergely) [inline ospo-book/content/en/04-chapter.md:112]:
```suggestion
> 2. Automation Security Checks: Implement automated security checks, such as the [OpenSSF Scorecard](https://scorecard.dev/), to continuously evaluate the security posture of projects. This proactive measure can highlight vulnerabilities or anomalies that merit further investigation.
```

File: ospo-book/content/en/04-chapter.md [hunk 5]
```diff
@@ -53,38 +50,88 @@ across various organizational divisions to align both technical and non-technica
...
+
+### Scenario #12
 
-- Recommendation:
+OSPOs face the challenge of navigating license changes and assessing software trustworthiness. As projects like Redis have shown, altering license terms can have significant implications for use, distribution, and contribution. These changes needs clear communication and understanding of the roles and responsibilities dictated by new license terms. Furthermore, OSPOs are tasked with evaluating the trustworthiness of software, which can vary based on whether a project is maintained by a single vendor or hosted under a foundation. For instance, The Almalinux Foundation presents a case where donating a project to a foundation mitigated risks associated with single-vendor governance, thereby enhancing trust in the project.
+
+> Recommendation
+>
+> 1. Educational Initiatives on License Implications: Develop educational materials and sessions for developers and users within the organization to understand the nuances of different licenses. This understanding will help them make informed decisions when using or contributing to open-source projects.
```
Comment (CsatariGergely) [inline ospo-book/content/en/04-chapter.md:126]:
```suggestion
> 1. Educational Initiatives on License Implications: Develop educational materials and sessions for developers and users within the organization to understand the nuances of different licenses and open source governance models. This understanding will help them make informed decisions when using or contributing to open-source projects.
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Comment by alice-sowerby** [comment]
@anajsana @CsatariGergely do you need anything from me / can I help in any way?

**Comment by alice-sowerby** [comment]
Hi @anajsana @todogroup/book-reviewers , sorry for the delay in getting to review this. I have changed my notification settings so that hopefully I get notified when I am tagged :( 

I think this chapter needs a full copy edit and I can't get a the GH interface to let me do that in this PR. That could be my lack of experience. 

How would I go about getting the file from the branch to edit it?

**Comment by alice-sowerby** [comment]
> Hi @anajsana @todogroup/book-reviewers , sorry for the delay in getting to review this. I have changed my notification settings so that hopefully I get notified when I am tagged :(
> 
> I think this chapter needs a full copy edit and I can't get a the GH interface to let me do that in this PR. That could be my lack of experience.
> 
> I will get a copy of the file from the branch to edit it and send back.

@anajsana I have edited a copy of the file locally, do you want me to create a PR?

**Comment by anajsana** [comment]
> > Hi @anajsana @todogroup/book-reviewers , sorry for the delay in getting to review this. I have changed my notification settings so that hopefully I get notified when I am tagged :(
> 
> > 
> 
> > I think this chapter needs a full copy edit and I can't get a the GH interface to let me do that in this PR. That could be my lack of experience.
> 
> > 
> 
> > I will get a copy of the file from the branch to edit it and send back.
> 
> 
> 
> @anajsana I have edited a copy of the file locally, do you want me to create a PR?

Thank you @alice-sowerby ! If you could add the copy edit changes to this PR instead of creating a new one, it will be awesome. Just let me know if you need help!

**Comment by alice-sowerby** [comment]
@anajsana I will see if I can figure out how to do it, and let you know if I hit any problems - thank you!

**Comment by anajsana** [comment]
@alice-sowerby created a new PR with these changes + copy edits. See: https://github.com/todogroup/ospology/pull/503

Tag this as duplicated and Closing with no merge.

**Pull request closed without merging**